modules/authentication.js

```diff
@@ -19,6 +19,11 @@ const onLogin = async (email, password, dispatch) => {
         loginMessage: `Welcome ${response.data.name}`,
         logoutMessage: "",
         showLoginForm: false,
+        communityId: response.data.community_id,
+        communityStatus: response.data.community_status,
+        phone: response.data.phone_number,
+        userAddress: response.data.address,
+        aboutMe: response.data.about_me,
       },
```

Comment:
i think we should remove the phone, useradress and about me from here, since we are setting that in the function where we get the profile information